### PR TITLE
Add $.isNumeric

### PR DESCRIPTION
**Pull request by Richienb**
Add documentation for `$.isNumeric()`.

### CODE REVIEW

**Comment by FranDias** [comment]
Hey, thanks for adding these docs. I'm happy to merge so you can get github rep but it won't appear live on the site https://github.com/HubSpot/youmightnotneedjquery/issues/200

**Comment by Abbondanzo** [comment]
Hey, I'd be happy to merge this and #207 #206 #205 #204 if you adjust the files to the new repo structure

File: comparisons/utils/is_numeric/ie8.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function isNumeric(val) {
```
Comment (Abbondanzo) [inline comparisons/utils/is_numeric/ie8.js]: The function uses `num` everywhere as the checked parameter, not `val`

```suggestion
function isNumeric(num) {    
```

**Comment by Richienb** [comment]
@FranDias

**Review by Abbondanzo — APPROVED** [review]

**Pull request merged**